src/display/font_loader.js

```diff
@@ -71,6 +78,37 @@ class FontLoader {
     }
   }
 
+  async loadFont(args) {
```

Comment:
Given that this code shouldn't be reached if `disableFontFace=true` is set, as far as I can tell, can we please assert that?
```suggestion
  async loadFont(args) {
    assert(!this.disableFontFace, "loadFont shouldn't be called when `disableFontFace` is set.")
```